src/os_posix/os_fallocate.c

```diff
@@ -18,6 +18,8 @@
  *	Call the underlying fallocate function, wrapped in a macro so there's only a single copy of
  * the mmap support code.
  */
+#if defined(HAVE_FALLOCATE) || (defined(__linux__) && defined(SYS_fallocate)) || \
+  defined(HAVE_POSIX_FALLOCATE)
```

Comment:
Looks like this is to support mmap. Do we have a build flag for enabling mmap?  We can use it as the condition instead.

Reply:
I believe it's a configuration option not a build flag.